source/BreachModel.h

```diff
@@ -14,8 +14,12 @@ class BreachModel {
 	int player;
 	/** Reference to image in SceneGraph for animation */
 	std::shared_ptr<cugl::PolygonNode> sprite;
+	/** Set to true if sprite needs to be updated */
+	bool needSpriteUpdate;
 
    public:
+	/** Default Max Health of a Breach*/
+	const unsigned int HEALTH_DEFAULT = 3;
```

Comment:
nit: Should probably be a `constexpr` instead of `const`.

Reply:
`Constexpr` needs to be instantiated at compile time, `const` can be at run-time. Any reason we're using `constexpr`?

Reply:
The fact that it's a compile-time constant saves a lookup at runtime. It's a replacement for `#define` macros.